global.json

```diff
@@ -1,3 +1,6 @@
 {
-    "projects": ["src", "test/WebSites", "samples"]
+    "projects": ["src", "test/WebSites", "samples"],
+    "sdk": {
+      "version": "1.0.0-preview2-003154"
```

Comment:
This is the SDK version the build downloads. Is it the correct one for the 1.0.3 milestone?